packages/react-components/src/components/ChatMessage/MyMessageComponents/ChatMyMessageComponentAsMessageBubble.tsx

```diff
@@ -261,7 +265,10 @@ const MessageBubble = (props: ChatMyMessageComponentAsMessageBubbleProps): JSX.E
           data-ui-id="chat-composite-message"
           author={<Text className={chatMessageDateStyle}>{message.senderDisplayName}</Text>}
           timestamp={
-            <Text className={chatMessageDateStyle} data-ui-id={ids.messageTimestamp}>
+            <Text
+              className={props.message.status === 'failed' ? chatMessageDateFailedStyle(theme) : chatMessageDateStyle}
```

Comment:
We also have this style used in `ChatMessageComponentAsMessageBubble`. Should we change the styles there too?

Reply:
what's the difference between ChatMessageComponentAsMessageBubble and ChatMyMessageComponentAsMessageBubble?